wandb/sdk/launch/templates/_wandb_bootstrap.py

```diff
@@ -1,10 +1,9 @@
 import json
-import multiprocessing
 import os
 import subprocess
 import sys
 
-CORES = multiprocessing.cpu_count()
+CORES = 1
```

Comment:
Curious why we got rid of this optimization?

Reply:
There's a tricky bug in the fail loop that I was having a hard time figuring out. If a package fails to install because it does not exist packages following it in the list to be installed would fail to install. The format of the output stream wasn't consistent between running on my machine, or during the docker build. I've filed a ticket for that issue to bring back this optimization, but given time constraints it was easiest to pull it out and revisit it.